core/domain/learner_progress_services_test.py

```diff
@@ -1046,6 +1058,42 @@ def test_remove_topic_from_learnt_list(self) -> None:
         self.assertEqual(self._get_all_learnt_topic_ids(
             self.user_id), [])
 
+    def test_remove_collection_from_completed_list(self) -> None:
+        self.assertEqual(self._get_all_completed_collection_ids(
+            self.user_id), [])
+
+        # Cannot remove a collection from non exisitng CompletedActivitiesModel.
+        learner_progress_services.remove_collection_from_completed_list(
+            self.user_id, self.COL_ID_0)
+        self.assertEqual(self._get_all_completed_collection_ids(
+            self.user_id), [])
+
+        # Add two collections to the completed list.
+        learner_progress_services.mark_collection_as_completed(
+            self.user_id, self.COL_ID_0)
+        learner_progress_services.mark_collection_as_completed(
+            self.user_id, self.COL_ID_1)
+        self.assertEqual(self._get_all_completed_collection_ids(
+            self.user_id), [self.COL_ID_0, self.COL_ID_1])
```

Comment:
```suggestion
        self.assertEqual(
            self._get_all_completed_collection_ids(self.user_id), 
            [self.COL_ID_0, self.COL_ID_1])
```

I think this indentation is clearer. ditto elsewhere

Reply:
Done